config/open-feature/org/workgroup.yaml

```diff
@@ -19,6 +19,19 @@ approvers:
   - benjiro
   - kinyoklion
   - weyert
+  - aepfli
+  - bacherfl
+  - benjiro
+  - davejohnston
+  - james-milligan
+  - josecolella
+  - Kavindu-Dodan
+  - kinyoklion
+  - staceypotter
```

Comment:
Community relationships / website / mailing list. Also helped with KubeCon prep and materials.